database/sqlite/sqlite_health.c

```diff
@@ -1090,3 +1151,202 @@ int alert_hash_and_store_config(
 
     return 1;
 }
+
+int sql_health_get_last_executed_event(RRDHOST *host, ALARM_ENTRY *ae, RRDCALC_STATUS *last_executed_status)
+{
+    int rc = 0, ret = -1;
+    char command[MAX_HEALTH_SQL_SIZE + 1];
+
+    char uuid_str[GUID_LEN + 1];
+    uuid_unparse_lower_fix(&host->host_uuid, uuid_str);
+
+    sqlite3_stmt *res = NULL;
+
+    snprintfz(command, MAX_HEALTH_SQL_SIZE, "select new_status from health_log_%s where alarm_id = %u and unique_id != %u and flags & %d order by unique_id desc LIMIT 1", uuid_str, ae->alarm_id, ae->unique_id, HEALTH_ENTRY_FLAG_EXEC_RUN);
```

Comment:
We have `define` with queries, and here we are passing direct as argument. I suggest to use one unique pattern for all.